timesketch/frontend/src/components/Sigma/SigmaList.vue

```diff
@@ -127,14 +186,167 @@ export default {
       return this.$store.state.meta
     },
   },
+  created() {
+    if (this.timeline) {
+      ApiClient.getSketchTimelineAnalysis(this.sketch.id, this.timeline.id)
+        .then(response => {
+          this.analyses = response.data.objects[0]
+        })
+        .catch(e => { })
+    }
+    // If no timeline was specified then loop over all of them
+    else {
+      this.sketch.timelines.forEach(timeline => {
+        ApiClient.getSketchTimelineAnalysis(this.sketch.id, timeline.id)
+          .then(response => {
+            this.analyses = this.analyses.concat(response.data.objects[0])
+          })
+          .catch(e => { })
```

Comment:
Given that you're already catching the error, might as well log / display something?

Reply:
https://github.com/google/timesketch/issues/2357